packages/dapp-svelte-wallet/_agstate/agoric-servers/package.json

```diff
@@ -12,6 +12,6 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "dependencies": {
-    "@agoric/cosmic-swingset": "^0.19.0"
+    "@agoric/vats": "^0.1.0"
```

Comment:
I'm nervous about having this temporary package name escape into the wild, and using it from dapps is the first step down that dark path. I don't know there's anything better we can do at this stage, though. Is there any way to avoid publishing something as generic as "`@agoric/vats`" to NPM? Or should I just try not to worry about this?

Reply:
I chose `vats` because that letter of the filename-completion alphabet was unused, and it really is "all the swingset vats that aren't ag-solo-specific".  It's only needed for `dapp-pegasus` and the wallet, not dapps in general.

I prefer `@agoric/vats` over many separate packages, and it's better than not separating them out at all.